schemas/jetstream/api/v1/stream_info_response.json

```diff
@@ -39,6 +39,11 @@
                   "pattern": "^[^.*>]*$",
                   "minLength": 0
                 },
+                "description": {
+                  "description": "A short description of the purpose of this stream",
```

Comment:
LGTM, not sure why these lines are here since they are already in main, anyway good to merge.